File: Anul II/Medii de proiectare si programare/Laborator/lab09/networking/src/main/java/com/ubb/protocol/request/CreateParticipantRequest.java
package com.ubb.protocol.request;

import com.ubb.model.Participant;

public class CreateParticipantRequest implements Request {
    private final Participant participant;

    public CreateParticipantRequest(Participant participant) {
        this.participant = participant;
    }

    public Participant getParticipant() {
        return participant;
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab08/C#/Ubb.BikeContest.Networking/Protocol/Request/CreateParticipantRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ubb.BikeContest.Model;

namespace Ubb.BikeContest.Networking.Protocol.Request
{
    [Serializable]
    public class CreateParticipantRequest : IRequest
    {
        private readonly Participant participant;

        public CreateParticipantRequest(Participant participant)
        {
            this.participant = participant;
        }

        public virtual Participant Participant
        {
            get { return participant; }
        }
    }
}
